Actions.js

```diff
@@ -0,0 +1,98 @@
+const express= require('express')
+const db=require('./data/helpers/actionModel')
+const router = express.Router();
+
+router.get('/', (req, res) => {
+    db.get()
+     .then(actions => {
+            res.status(200).json(actions)
+          }
+     )
+    .catch(err => {
+        res.status(500).json({
+          message: 'Could not retrieve actions.',
+          err
+        })
+    })
+  })
+
+router.get('/:id', (req, res) => {
+    const { id } = req.params;
+    db.get( id )
+
+     .then(action => {
+            res.status(200).json(action)
+          }
+     )
```

Comment:
what happens if you search for an action with an id that doesn't exist.  What would happen?